### PR TITLE
Ensure latest version from manifest is value for player_version in co…

### PR DESCRIPTION
**Pull request by stulees**
…nfig logger

### CODE REVIEW

File: src/main/loggers/config-logger.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ module.exports = {
         installer_version: commonConfig.getModuleVersion("launcher"),
         player_name: "RisePlayerElectron",
         java_version: "",
-        player_version: commonConfig.getModuleVersion(config.moduleName),
+        player_version: commonConfig.getLatestVersionInManifest(),
```
Comment (jungeunyoon) [inline src/main/loggers/config-logger.js]: Thoughts on using player version as a fallback?

Comment (stulees) [reply src/main/loggers/config-logger.js]: @jungeunyoon If get latest returns "", the local manifest doesn't contain any modules with valid version prop/value (or there was a failure in reading manifest), so falling back on trying to get players module version from manifest won't work either. Cheers

Comment (jungeunyoon) [reply src/main/loggers/config-logger.js]: @stulees Ah yes, you're correct. Thanks!

Comment (santiagonoguez) [reply src/main/loggers/config-logger.js]: @stulees is there a valid use case where getModuleVersion() for player returns something that does not start with "20" ? in that case the fallback may be useful.

Comment (stulees) [reply src/main/loggers/config-logger.js]: @santiagonoguez Not that I know of. @tejohnso Thoughts?

Comment (santiagonoguez) [reply src/main/loggers/config-logger.js]: if it exists, it would be some testing scenario, not  production of course

Comment (stulees) [reply src/main/loggers/config-logger.js]: @santiagonoguez Yeah we don't need to concern testing scenarios in this case as latest version is for display app and support.

Comment (jungeunyoon) [reply src/main/loggers/config-logger.js]: We should be good. I cant think of a case either unless it's the next millennium.

**Pull request merged**